wire-ios-data-model/Source/Universal backup/Model/ConversationBackupModel.swift

```diff
@@ -18,22 +18,35 @@
 
 import Foundation
 
-public struct ConversationBackupModel {
+public struct ConversationBackupModel: Codable {
 
-    public let qualifiedID: QualifiedID
+    public let id: UUID
 
-    public let name: String
+    public let domain: String
 
-    public let participants: Set<QualifiedID>
+    public let name: String?
```

Comment:
Is there a difference between an empty string and nil?

Reply:
We made it nil to fix the decoding errors, but it doesn't matter since we'll remove the name entirely.